vizro-core/docs/pages/user-guides/run.md

```diff
@@ -34,6 +34,27 @@ Dash is running on http://127.0.0.1:8050/
  * Debug mode: on
 ```
 
+??? info "Automatic reloading and debugging"
+
+    #### Reloading the dashboard
+
+    You can set up the front-end to automatically refresh whenever dashboard configuration updates are made, as described in the
+    ["Code Reloading and Hot Reloading" section of the Dash Dev Tools documentation](https://dash.plotly.com/devtools#code-reloading-&-hot-reloading).
+    It is turned off by default in Vizro apps but can be enabled by using `debug=True` in the `run()` method, eg.
+
+    `Vizro().build(dashboard).run(debug=True)`
+
+    (Note: this is the equivalent of enabling Dash Dev Tools via the `app.run()` method. See the [Dash documentation](https://dash.plotly.com/devtools) for more information).
+
+    #### Debugging
+
+    Dash dev tools debug mode is turned on as described above, by using `debug=True` in the `run()` method.
+
+    `Vizro().build(dashboard).run(debug=True)`
+
```

Comment:
```suggestion
```

I'd suggest simplifying this by just adding a bit more detail to the above. At the moment it seems too repetitive since this just repeats the above section.